app/views/govuk_publishing_components/components/_tabs.html.erb

```diff
@@ -1,5 +1,9 @@
 <%
   tabs ||= []
+  no_panel_border ||= false
```

Comment:
I think it would be better to use "panel_border" and set it to true by default - the double negative here is a bit confusing.